spec/classes/init_spec.rb

```diff
@@ -569,7 +598,7 @@
           v[:facts_hash]
         end
 
-        if v[:facts_hash][:os]['name'] == 'RedHat'
+        if v[:facts_hash][:os]['name'] == 'RedHat' and  v[:facts_hash][:os]['release']['major'] <= '7'
```

Comment:
You used `< 8` previously. Should stick with that for consistency.

Reply:
done